data_registry/process_manager/task/process.py

```diff
@@ -8,6 +8,8 @@
 
 logger = logging.getLogger(__name__)
 
+base_url = "/api/collections"
```

Comment:
Instead of extracting only the base URL, can do:

```python
def _url(path):
    return urljoin(settings.KINGFISHER_PROCESS_URL, f"/api/collections/{path}")
```